### PR TITLE
Use tlRelease instead of release

### PR DESCRIPTION
**Pull request by rossabaker**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -143,7 +143,7 @@ inThisBuild(
     githubWorkflowBuild +=
       WorkflowStep.Sbt(List(s"++$Scala213", "site/makeMicrosite"), cond = Some(Scala213Cond)),
     githubWorkflowPublish := Seq(
-      WorkflowStep.Sbt(List("release")),
+      WorkflowStep.Sbt(List("tlRelease")),
```
Comment (armanbilge) [inline build.sbt:146]: Should this be a `+=` instead?

**Comment by rossabaker** [comment]
Uh, probably.

File: build.sbt (hunk 1 above)
Comment (rossabaker) [reply build.sbt:146]: ~~Uh, probably.~~

Well, I don't know.  sbt-spiewak just sets it to `release`, overriding `publish`.  I guess it depends what `tlRelease` does, but it appears to already be an alias over publish.

Comment (armanbilge) [reply build.sbt:146]: `tlRelease` is sbt-spiewak's `release` with some slight modification to do snapshots properly.

Comment (armanbilge) [reply build.sbt:146]: You could also consider moving these steps to `githubWorkflowPublishPostamble`. But in general overriding the steps generated by sbt-typelevel should be (mostly) an anti-pattern.

**Pull request closed without merging**